content/francais/python-turtle/activity-3.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Activité 3 - Dessiner un hexagone"
+date: 2024-09-17T13:24:17-07:00
+weight: 5
+draft: false
+---
+
+![Turtle, hexagon animation](https://media.giphy.com/media/TDLqC61A5uOAO8WwnJ/giphy.gif)
```

Comment:
Turtle, hexagon animation is alt text that should be translated.. but this is tricky because the previous activity had something within the [] that shouldn't be translated...

We should have a user guide of common pitfalls of the AI tool and note them for users to review